packages/fxa-payments-server/src/routes/Product/SubscriptionRedirect/index.tsx

```diff
@@ -24,26 +24,36 @@ export const SubscriptionRedirect = ({ plan }: SubscriptionRedirectProps) => {
     downloadURL || productRedirectURLs[product_id] || defaultProductRedirectURL;
 
   useEffect(() => {
-    navigateToUrl(redirectUrl);
-  }, [navigateToUrl, redirectUrl]);
+    window.addEventListener('message', handleIframeTask);
+  });
+
+  const handleIframeTask = (e: any) => {
+    if (e.data === 'submitted survey') {
+      console.log('survey: ' + e.data);
+      navigateToUrl(redirectUrl);
+    }
+  };
 
   return (
     <div className="product-payment" data-testid="subscription-redirect">
       <div className="subscription-ready">
-        <h2>Your subscription is ready</h2>
-        <img
```

Comment:
This is [causing some test failures](https://circleci.com/gh/mozilla/fxa/127687?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link), now that the product icon & name are gone from the view.

Need to update / tweak these tests:

```
FAIL src/routes/Product/index.test.tsx
  routes/Product
    with product icon defined
      ✕ handles a successful payment submission as expected (113ms)
      ✕ redirects to product page if user is already subscribed (44ms)
    with default product icon
      ✕ handles a successful payment submission as expected (256ms)
      ✕ redirects to product page if user is already subscribed (43ms)

FAIL src/routes/Product/SubscriptionRedirect/index.test.tsx
  ✕ performs a redirect to the expected URL for local product (13ms)
  ✕ performs a redirect to the default URL for unknown product (15ms)
```

Removing icon & name seems consistent with the mock in #3456, but I wonder if we'll want to add them back later or conditionally?

Reply:
I think it is safe to assume that we would want to add them back at some point, but for now they won't be there.  How do you want me to handle this?  Keep the code and comment it out or remove it?

Reply:
We shouldn't leave code commented out - we can always get it back from previous commits. Might as well remove the bits that are failing. (i.e. not the entire tests, just the parts that depend on product name & icon being available)

Reply:
OK I removed the tests that were causing error.